src/shared/ui/select/Select.tsx

```diff
@@ -32,11 +20,35 @@ export const Select = ({ label, placeholder = label, items, onSelect }: SelectPr
 
```

Comment:
Just a question - Why is the anchor for? I mean: what do you achieve with this? You're checking if menu is expanded and then, according to the isMenuOpen variable, setting appropriate css class? Could it be done with simple state that holds true/false value?